touchforms/backend/xformplayer.py

```diff
@@ -216,6 +241,44 @@ def walk(self):
         self._walk(form_ix, tree)
         return tree
 
+
+    def build_element_tree(self, javarosa_node, element_tree_node):
+        """
+        :param javarosa_node: the element tree node to expand into
+        :param element_tree_node: the Javarosa tree ref to expand form
+        :return: the expanded element tree node with all children and attributes
+        from the javarosa node
```

Comment:
haha, it has a good doc string and i still have no idea what this function does.

also are the param definitions backwards or just confusing?